app/helpers.php

```diff
@@ -68,14 +68,10 @@ function authenticatedOnSSOGuard(): bool
  */
 function reportException($exception, $message = null): void
```

Comment:
worth specifying this will be a string? Could mean you want to pass in a throwable